pkg/provisioner/crdprovisioner.go

```diff
@@ -1217,14 +1245,25 @@ func waitForDetachVolumeFunc(obj interface{}, objectDeleted bool) (bool, error)
 	return false, nil
 }
 
-func startWatchingKube(ctx context.Context, kubeInformerFactory informers.SharedInformerFactory) {
+// Initializing nodeInformer and crdInformer and waiting for cache sync
+func startWatchingObjs(ctx context.Context, kubeInformerFactory informers.SharedInformerFactory, crdInformerFactory crdInformers.SharedInformerFactory, driverUninstallState *uint32) {
```

Comment:
Consider making `startWatchingObjs` a method of `crdProvisioner` to avoid passing the informers and install state as arguments.